generators/cucumber/templates/env.rb

```diff
@@ -1,9 +1,11 @@
 require 'cucumber'
+require 'howitzer/utils/transformers'
```

Comment:
Why did you migrated transformers onto library side? It was in generators. There were 2 motivations to do like that:
1 - generated code can be customized on project-level side.
2 - the code is related with Cucumber only and is using in Cucumber based projects